imap_processing/glows/l1a/glows_l1a.py

```diff
@@ -118,19 +157,19 @@ def generate_de_dataset(
     """
     # TODO: Block header per second, or global attribute?
 
+    # Store timestamps for each DirectEventL1a object.
     time_data = np.zeros(len(de_l1a_list), dtype="datetime64[ns]")
     # TODO: Should each timestamp point to a list of direct events, each with a
     #  timestamp? Or should the list be split out to make the timestamps?
 
+    # Create a 3D array to store the direct events data
     # Each DirectEventL1A class covers 1 second of direct events data
     direct_events = np.zeros((len(de_l1a_list), len(de_l1a_list[0].direct_events), 4))
-    global_attributes = glows_cdf_attrs.glows_l1a_de_attrs.output()
 
-    global_attributes["ground_software_version"] = __version__
-    global_attributes["Data_version"] = data_version
-    # In header: block header, missing seqs
-    # Time varying - statusdata
+    imap_object.add_global_attribute("Data_version", data_version)
+    # TODO: What is ground_software_version??
```

Comment:
Oh yeah, this also came up in another comment. This would mean the version of `imap_processing`, which now is automatically added by some changes Greg made. So it's no longer necessary here.